pontoon/base/migrations/0029_external_accounts.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.13 on 2022-08-03 19:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("base", "0028_userprofile_new_contributor_notifications"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="userprofile",
+            name="bugzilla",
+            field=models.EmailField(
+                blank=True,
+                max_length=254,
```

Comment:
```suggestion
                max_length=255,
```

254 is a bit of a weird length, any particular reason?

Reply:
That is auto generated and implicitly set by EmailField:
https://docs.djangoproject.com/en/4.0/ref/models/fields/#emailfield

Here's the background:
https://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690